src/main/java/baron/tasks/Task.java

```diff
@@ -29,10 +54,21 @@ public boolean unmark() {
         return false;
     }
 
+    /**
+     * Returns the string representation of this task.
+     *
+     * @return the string representation of this task.
+     */
     public String toString() {
         return "[" + this.getStatusIcon() + "] " + description;
     }
 
+    /**
+     * Returns the encoded string to be stored in the storage using the specified delimiter.
+     *
+     * @param delimiter the string to be inserted in between various information of this task.
+     * @return the encoded string to be stored in the storage using the specified delimiter.
+     */
     public String toSaveString(String delimiter) {
```

Comment:
I believe ``saveString`` is sufficient.

Reply:
Hi De Shao, thank you for the suggestion, but I feel that as this method doesn't save string, so `saveString` would be misleading (it may mean that this method saves the string somewhere, `save` as in the verb). In fact, it just returns the encoded string that is to be saved somewhere by other parts of my code.